ch00python/028dictionaries.ipynb

```diff
@@ -390,33 +349,31 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### No guarantee of order"
+    "### No guarantee of order (before python 3.6)"
```

Comment:
```suggestion
    "### No guarantee of order (before Python 3.7)"
```

Reply:
In the CPython is from 3.6, in the language is 3.7. But yeah, this will make it less confusing.